runtime/src/ctype/tests.rs

```diff
@@ -18,68 +18,96 @@
 
 use super::*;
 
-use primitives::{Blake2Hasher, H256};
-use runtime_io::with_externalities;
-use runtime_primitives::{
-	testing::{Digest, DigestItem, Header},
-	traits::{BlakeTwo256, IdentityLookup},
-	BuildStorage,
+use crate::Signature;
+use sp_core::{ed25519, Pair, H256};
+use sp_runtime::{
+    testing::Header,
+    traits::{BlakeTwo256, IdentifyAccount, IdentityLookup, Saturating, Verify},
+    MultiSigner, Perbill,
+};
+use support::{
+    assert_err, assert_ok, impl_outer_origin, parameter_types,
+    weights::{
+        constants::{BlockExecutionWeight, ExtrinsicBaseWeight, RocksDbWeight},
+        Weight,
+    },
 };
-use support::{assert_err, assert_ok, impl_outer_origin};
 
 impl_outer_origin! {
-	pub enum Origin for Test {}
+    pub enum Origin for Test {}
 }
 
-#[derive(Clone, Eq, PartialEq)]
+#[derive(Clone, Eq, PartialEq, Debug)]
 pub struct Test;
+
+parameter_types! {
+    pub const BlockHashCount: u64 = 250;
+    pub const MaximumBlockWeight: Weight = 1_000_000_000;
+    pub const MaximumBlockLength: u32 = 5 * 1024 * 1024;
+    pub const AvailableBlockRatio: Perbill = Perbill::from_percent(75);
+    // FIXME: Wait for workaround
```

Comment:
Placeholder to be aware of this.

Reply:
Fixed in 466ab19 by removing `const` declaration from `MaximumExtrinsicWeight` 😅